tests/behaviour/steps/qbwriter.py

```diff
@@ -895,13 +872,53 @@ def assert_uri_style_for_uri(uri_style: URIStyle, uri: str, node):
             ".json"
         ), f"expected {node} to end with .csv or .json"
 
-@then(u'the RDF should contain version specific triples')
+
+@then("the RDF should contain version specific triples")
 def step_impl(context):
-    version_triples = (context.text + f" prov:used <{get_csvcubed_version_uri()}> .").strip()
-    version_triples = re.sub("\r", "", version_triples) # windows problem   
+    version_triples = (
+        context.text + f" prov:used <{get_csvcubed_version_uri()}> ."
+    ).strip()
+    version_triples = re.sub("\r", "", version_triples)  # windows problem
 
     version_triples_graph = Graph().parse(format="turtle", data=version_triples)
     test_graph_diff(
         Graph().parse(format="turtle", data=context.turtle),
         version_triples_graph,
     )
+
+
+@Given('a pivoted shape cube with identifier "{identifier}" named "{cube_name}"')
+def step_impl(context, identifier: str, cube_name: str):
+    metadata = CatalogMetadata(title=cube_name, identifier=identifier)
+    data = pd.DataFrame(
+        {
+            "Some Dimension": ["a", "b", "c"],
+            "Some Attribute": ["attr-a", "attr-b", "attr-c"],
+            "Some Obs Val": [1, 2, 3],
+            "Some Other Obs Val": [2, 4, 6],
+        }
+    )
+    columns = [
+        QbColumn("Some Dimension", NewQbDimension.from_data("Some Dimension", data["Some Dimension"])),
+        QbColumn("Some Attribute", NewQbAttribute.from_data("Some Attribute", data["Some Attribute"], observed_value_col_title="Some Obs Val")),
+        QbColumn(
+            "Some Obs Val",
+            QbObservationValue(NewQbMeasure("Some Measure"), NewQbUnit("Some Unit")),
+        ),
+        QbColumn(
+            "Some Other Obs Val",
+            QbObservationValue(NewQbMeasure("Some Other Measure"), NewQbUnit("Some Unit")),
+        ),
+    ]
+
+    cube = Cube(metadata=metadata, data=data, columns=columns)
+    context.cube = cube
+
+@Given('the environment variable "{env_var_name}" is "{env_var_value}"')
```

Comment:
Can we just make it accept a string as the `env_var_value`? That'd give us more flexibility.